tests/acceptance/stepDefinitions/publicLinkContext.js

```diff
@@ -163,6 +163,29 @@ When(
     await client.page.FilesPageElement.publicLinksDialog().addPassword(linkName, password)
   }
 )
+When(
+  'the user opens the link edit dialog of file/folder/resource {string} with name {string} using the webUI',
```

Comment:
Could you check because there's another definition in `the user renames the public link named {string} of file/folder/resource {string} to {string}` and remove/unite them?

Reply:
@pascalwengerter 
did you mean this one? 
https://github.com/owncloud/web/blob/83373596d2d7fea06d916ae296392b3b2d2a5b5e/tests/acceptance/stepDefinitions/publicLinkContext.js#L190-L208

or rename one
https://github.com/owncloud/web/blob/83373596d2d7fea06d916ae296392b3b2d2a5b5e/tests/acceptance/stepDefinitions/publicLinkContext.js#L120-L124

rename one looks okay to me